CHANGELOG.md

```diff
@@ -2,8 +2,16 @@ Change Log
 ==========
 
 ## Next Version
+- Various bug fixes.
 
-- Bug fixes to improve local development.
+## Version 1.1.0
```

Comment:
This is the expected new version, right?

Reply:
Yes correct, after six weeks, you are right to increment this one.